### PR TITLE
Add RSS Feed Formatter Script

### PR DESCRIPTION
**Pull request by amanifarooque**
This script takes a txt file with RSS feed URLs and outputs the links into a single txt file for bulk upload.

### CODE REVIEW

File: team/indexing-scripts/feed_formatter.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import feedparser
+
+## Instructions
+## 1. Create "feeds.txt" file in the "indexing-scripts" folder. Ensure it has all RSS feeds you want to process, with each feed on a new line. 
+## 2. Ensure all feeds in the file are valid. The script will error if there are spaces in the url, so find-replace " " with "%20".
+## 3. Open Terminal, and cd into the /indexing-scripts folder
+## 4. Run `python3.9 python_formatter.py`
```
Comment (dawnpm) [inline team/indexing-scripts/feed_formatter.py]: feed_formatter.py or python_formatter.py?

**Review by dawnpm — CHANGES_REQUESTED** [review]
one question about file name vs instructions - thanks!

File: team/indexing-scripts/feed_formatter.py (hunk 1 above)
Comment (amanifarooque) [reply team/indexing-scripts/feed_formatter.py]: Fixed!

**Comment by amanifarooque** [comment]
@dawnpm Requesting re-review, thanks!

**Review by dawnpm — APPROVED** [review]

**Pull request merged**